about.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/baf7e9f545.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
+    <title>Hum Style Awards</title>
+</head>
+
+<body>
+    <header>
+        <div class="hidden md:flex items-center justify-end h-10" id="top-header">
+            <div class="icons space-x-4 pr-6">
+                <a href="#" rel="noopener"><i class="fa-brands fa-facebook-f text-base"></i></a>
+                <a href="#" rel="noopener"><i class="fa-brands fa-twitter text-base"></i></a>
+                <a href="#" rel="noopener" class="hover:underline text-base">English</a>
+            </div>
+        </div>
+        <nav class="h-20 shadow-md flex items-center justify-between">
+            <!-- hamburger menu -->
+            <div class="md:hidden pl-12" id="btn-menu">
+                <button id="menu-btn" type="button" class="z-40 block hamburger md:hidden focus:outline-none">
+                    <span class="hamburger-top"></span>
+                    <span class="hamburger-middle"></span>
+                    <span class="hamburger-bottom"></span>
+                </button>
+            </div>
+            <div class="logo pr-12 md:pl-24">
+                <a href="index.html" class="text-2xl font-bold">Hum Style Awards</a>
+            </div>
+            <div class="menu-links hidden md:flex md:items-center  space-x-6 pr-6 ">
+                <a href="about.html" class="text-lg " rel="noopener">About</a>
```

Comment:
- [ ] [OPTIONAL] You were supposed to create a Home button for the desktop screen on the about page but you didn't. I only marked it as optional because the previous reviewer missed it.